content/java/concepts/math-methods/terms/addExact/addExact.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '.addExact()'
+Description: 'Returns the sum of its arguments.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Function'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.addExact()`** method returns the sum of its arguments. It will throw an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows either `int` or `long`. This function is `static`, therefore object creation is not required.
```

Comment:
```suggestion
The **`Math.addExact()`** method returns the sum of its arguments. It will throw an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows either `int` or `long`.
```

I don't think the last sentence is necessary since all `java.lang.Math` methods are static.